testsuite/forge/src/runner.rs

```diff
@@ -43,7 +43,25 @@ pub struct Options {
         long,
         help = "If set, tries to connect to a local swarm instead of testnet"
     )]
-    local_swarm: bool,
+    pub local_swarm: bool,
+
+    // emit_tx options
+    #[structopt(long, default_value = "15")]
+    accounts_per_client: usize,
+    #[structopt(long)]
+    workers_per_ac: Option<usize>,
+    #[structopt(
+        long,
+        help = "Time to run --emit-tx for in seconds",
+        default_value = "60"
+    )]
+    duration: u64,
+
+    // TODO, need to remove these args once we can load from vault
+    #[structopt(long, default_value = "")]
+    pub root_key: String,
+    #[structopt(long, default_value = "")]
+    pub treasury_compliance_key: String,
```

Comment:
These need to not be apart of this Options struct. This struct is used to represent the arguments that are usually passed in to standard rust test harnesses. Instead we can create a different struct that holds both these by using `#[structopt(flatten)]` and then passing in the Options struct.